CMakeLists.txt

```diff
@@ -63,9 +63,9 @@ include(compiler)
 include(os)
 
 if (64_BIT)
-	set(ARCH "64bit")
+	set(ARCH "x64")
 else()
-	set(ARCH "32bit")
+	set(ARCH "x86")
 endif()
```

Comment:
In future we should find a way to detect the architecture properly (could be ARM instead of x86, for example).

Given that it's only a cosmetic issue (at least, for now), I would consider it low priority.

Reply:
Agreed. The build files have intelligence when it comes to those settings, so that shouldn't be an issue. It's only meant to work on `WIN32`, when `x86` or `x64`.